exec.go

```diff
@@ -24,7 +24,8 @@ type EXEC struct {
 
 // CommandOptions contains the options that can be passed to command.
 type CommandOptions struct {
-	Dir string
+	Dir  string
+	Cont bool
```

Comment:
Names longer than 4 characters don't seem to receive the value from javascript...(!)